src/lib/openapi/spec/create-user-schema.ts

```diff
@@ -0,0 +1,31 @@
+import { FromSchema } from 'json-schema-to-ts';
+
+export const createUserSchema = {
+    $id: '#/components/schemas/createUserSchema',
+    type: 'object',
+    additionalProperties: false,
+    required: ['rootRole'],
```

Comment:
we actually also require `username` or `email`. But that might not be possible to express in OAS?

Reply:
tbh, I think we should require email. That's also what we do require in the UI at the moment.